src/test/java/com/suse/saltstack/netapi/config/ClientConfigTest.java

```diff
@@ -31,5 +31,4 @@ public void testPutGetRemove() {
         config.remove(key);
         assertEquals("Should return the default value after removing the key", config.get(key), key.defaultValue);
```

Comment:
I think i will wrap after the description like so:
```java
assertEquals("Should return the default value after removing the key",
        config.get(key), key.defaultValue);
```
That way it fits the 92 limit and i think it looks nice having `expected` and `actual` value together. Is that ok with you?